### PR TITLE
Feature/implement api mypath

### PR DESCRIPTION
**Pull request by votheduy154**

### CODE REVIEW

File: src/containers/myPath/myPath.js [hunk 1]
```diff
@@ -139,48 +180,29 @@ class PathPage extends Component {
...
+                    </tr>
+                  )
+                }
               </tbody>
             </table>
             {/* End table */}
-            <nav aria-label="Page navigation example" className="pb-1">
+            {/* <nav aria-label="Page navigation example" className="pb-1">
               <ul className="pagination justify-content-center">
                 <li className="page-item active"><a className="page-link" href="">1</a></li>
```
Comment (nhatdev1193) [inline src/containers/myPath/myPath.js:193]: comment khuc nay la sao a ?

File: src/containers/myPath/myPath.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import * as Icon from 'react-icons/lib/fa/';
 import PropTypes from 'prop-types';
 import {Radar, RadarChart, PolarGrid, Legend, PolarAngleAxis, ResponsiveContainer, PolarRadiusAxis} from 'recharts';
 import CircularProgressbar from 'react-circular-progressbar';
+import * as _ from 'lodash';
 
 let data = [
   { subject: 'Skill 1', A: 120, B: 110, fullMark: 150 },
```
Comment (nhatdev1193) [inline src/containers/myPath/myPath.js]: cai data nay minh remove di a, get data tu api tra ve thi dung cai ham cua e de show skill

File: src/containers/myPath/myPath.container.js [hunk 3]
```diff
@@ -1,19 +1,41 @@
...
+    const {showLoading} = this.props.payload.initReducer
+    const {listInstrument, myCourses} = this.props.payload.pathReducer
+    if ((showLoading === true) || (listInstrument === undefined && myCourses === undefined)) {
+      return (
+        <Loading/>
+      )
+    } else {
+      return (
+        <MyPathPage
+          {...this.state}
```
Comment (nhatdev1193) [inline src/containers/myPath/myPath.container.js]: sao lai ``` {...this.state}} ```o day a?

**Review by nhatdev1193 — CHANGES_REQUESTED** [review]

File: src/containers/myPath/myPath.js (hunk 1 above)
Comment (votheduy154) [reply src/containers/myPath/myPath.js:193]: api chua du nen anh commnet de do

File: src/containers/myPath/myPath.container.js (hunk 3 above)
Comment (votheduy154) [reply src/containers/myPath/myPath.container.js]: cho nay co cai remove path, anh truyen redux vao state, khi remove thi no cap nhat trang thai delete luon, ko phai loading lai trang

File: src/containers/myPath/myPath.js (hunk 1 above)
Comment (nhatdev1193) [reply src/containers/myPath/myPath.js]: cai khuc nay la phan trang fai ko a? https://github.com/AdeleD/react-paginate a dung thang nay, api chi tra ve cai array da sort roi, minh se phan trang.

File: src/containers/myPath/myPath.container.js (hunk 3 above)
Comment (votheduy154) [reply src/containers/myPath/myPath.container.js]: ah anh nhin nham, bua de state o ngoai h chuyen vo trong, de anh xoa :))

File: src/containers/myPath/myPath.js (hunk 2 above)
Comment (votheduy154) [reply src/containers/myPath/myPath.js]: api chua fix em, dang doi fix

File: src/containers/myPath/myPath.js (hunk 1 above)
Comment (nhatdev1193) [reply src/containers/myPath/myPath.js:193]: cai nay phan trang fai ko a? a dung https://www.npmjs.com/package/react-paginate de phan trang, tai api tra ve cai array da sort thoi.

**Review by nhatdev1193 — APPROVED** [review]

**Pull request merged**